formfactory/models.py

```diff
@@ -133,9 +133,6 @@ class Meta(object):
     def __unicode__(self):
         return self.title
 
-    def absolute_url(self):
```

Comment:
That's your implementation of an API. Use get_absolute_url like all our other apps.